chain/network/src/peer_manager/peer_manager_actor.rs

```diff
@@ -61,8 +59,6 @@ const MONITOR_PEERS_INITIAL_DURATION: time::Duration = time::Duration::milliseco
 const BROADCAST_VALIDATED_EDGES_INTERVAL: time::Duration = time::Duration::milliseconds(50);
 /// Maximum amount of time spend processing edges.
 const BROAD_CAST_EDGES_MAX_WORK_ALLOWED: time::Duration = time::Duration::milliseconds(50);
-/// Delay syncinc for 1 second to avoid race condition
-const WAIT_FOR_SYNC_DELAY: time::Duration = time::Duration::milliseconds(1_000);
```

Comment:
I don't know what race condition it is referring to. There were some test failures, but due to poorly written tests, not because the logic is incorrect now (AFAICT).